src/neo/SmartContract/Native/TransactionState.cs

```diff
@@ -40,16 +46,19 @@ public class TransactionState : IInteroperable
         void IInteroperable.FromStackItem(StackItem stackItem)
         {
             Struct @struct = (Struct)stackItem;
-            BlockIndex = (uint)@struct[0].GetInteger();
-            _rawTransaction = @struct[1];
+            Trimmed = @struct[0].GetBoolean();
+            if (Trimmed) return;
```

Comment:
We shouldn't return any trimmed transactions from methods of native Ledger contract. In neo-go, we store hashes of conflicting transactions in a *trimmed* manner for easier further conflict checking. Any other attempts to get trimmed transaction should lead to error, see https://github.com/nspcc-dev/neo-go/blob/473d11d24e782a04c48d08c128df60007700ac70/pkg/core/dao/dao.go#L547.

Reply:
Ledger contract won't return trimmed transaction. see https://github.com/neo-ngd/neo/blob/fef1c064a05c8d7c529328155db2398d8b9e92b2/src/neo/SmartContract/Native/LedgerContract.cs#L229